src/App/utils/SearchBox/index.js

```diff
@@ -29,7 +29,7 @@ const SearchBox = ({ className, mobile }) => {
 
         return (
             <ul className="item-list item-list-hover">
-                {tempSearchResultList.map(directory => directory.map(result => <Link tabIndex="-1" onKeyDown={e => arrowNavigation(e)} className="res" key={result.path} onClick = {() => hideResultBox()} to={result.path}><li><span className="result" dangerouslySetInnerHTML={{ __html: modify(result.title, searchTerm) }}></span><span className="directory">{result.path.split("/")[1].charAt(0).toUpperCase() + result.path.split("/")[1].slice(1)}</span></li></Link>))
+                {tempSearchResultList.map(directory => directory.map(result => <Link tabIndex="0" onKeyDown={e => arrowNavigation(e)} className="res" key={result.path} onClick = {() => hideResultBox()} to={result.path}><li><span className="result" dangerouslySetInnerHTML={{ __html: modify(result.title, searchTerm) }}></span><span className="directory">{result.path.split("/")[1].charAt(0).toUpperCase() + result.path.split("/")[1].slice(1)}</span></li></Link>))
```

Comment:
Why was it necessary to change to 0? Do you remember? 😄

Reply:
To be able to tab through search results :)

Reply:
ah! It was intended, since the only possible keyboard nav was with arrows. When tabbing, you are "moving" away from the component as a whole :D

Reply:
Ahh, will redo the change :)